Source/Particles/Pusher/UpdatePosition.H

```diff
@@ -0,0 +1,28 @@
+#ifndef WARPX_PARTICLES_PUSHER_UPDATEPOSITION_H_
+#define WARPX_PARTICLES_PUSHER_UPDATEPOSITION_H_
+
+#include <AMReX_REAL.H>
```

Comment:
We need to include more headers because `AMREX_GPU_HOST_DEVICE` and `AMREX_INLINE` are not in `AMReX_REAL.H`.  I would replace this with `#include <AMReX_FArrayBox.H>`.  We also need to include header that knows `PhysConst`.